### PR TITLE
amp-winoff: Add version 5.0.1

### PR DESCRIPTION
**Pull request by wenmin92**

### CODE REVIEW

File: bucket/amp-winoff.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "5.0.1",
+    "description": "An utility for scheduling the shutdown of Windows computers, with several shutdown conditions and fully configurable.",
```
Comment (Ash258) [inline bucket/amp-winoff.json:3]:
```suggestion
    "description": "Fully configurable utility for scheduling the shutdown of Windows computers",
```

File: bucket/amp-winoff.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "shortcuts": [
+        [
+            "WinOFF.exe",
+            "AMP WinOFF"
+        ]
+    ],
+    "checkver": "Version ([\\d.]+)",
+    "autoupdate": {
+        "url": "http://www.ampsoft.net/files/WinOFF.zip"
+    }
```
Comment (Ash258) [inline bucket/amp-winoff.json:18]: Latest version was 2010

```suggestion
    ]
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/amp-winoff.json (hunk 2 above)
Comment (wenmin92) [reply bucket/amp-winoff.json]: Thanks!

**Review by Ash258 — APPROVED** [review]

**Pull request merged**